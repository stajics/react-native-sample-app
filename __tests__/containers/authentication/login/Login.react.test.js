import 'react-native';
import React from 'react';
import { Login } from '../../../../app/containers/authentication/login/Login.react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'; // eslint-disable-line

jest.mock('../../../../app/components/forms/LoginForm.react', () => 'LoginForm');

it('renders correctly', () => {
  const login = renderer.create(
    <Login />
  ).toJSON();
  expect(login).toMatchSnapshot();
});
