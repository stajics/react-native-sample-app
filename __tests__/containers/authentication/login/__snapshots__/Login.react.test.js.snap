exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "marginTop": 50,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 40,
        "fontWeight": "bold",
        "marginBottom": 70,
      }
    }>
    Sample App
  </Text>
  <LoginForm />
</View>
`;
